Source/Core/VideoBackends/D3D/TextureCache.cpp

```diff
@@ -87,16 +87,14 @@ void TextureCache::TCacheEntry::CopyRectangleFromTexture(
 	if (srcrect.GetWidth() == dstrect.GetWidth()
 		&& srcrect.GetHeight() == dstrect.GetHeight())
 	{
-		const D3D11_BOX *psrcbox = nullptr;
 		D3D11_BOX srcbox;
-		if (srcrect.left != 0 || srcrect.top != 0)
-		{
-			srcbox.left = srcrect.left;
-			srcbox.top = srcrect.top;
-			srcbox.right = srcrect.right;
-			srcbox.bottom = srcrect.bottom;
-			psrcbox = &srcbox;
-		}
+		srcbox.left = srcrect.left;
+		srcbox.top = srcrect.top;
+		srcbox.right = srcrect.right;
+		srcbox.bottom = srcrect.bottom;
+		srcbox.front = 0;
+		srcbox.back = 1;
```

Comment:
Is it possible for this code path to deal with EFB textures? In that case we should probably copy all layers.